### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by wojpawlik**
# Description

## Type of change

# How Has This Been Tested?

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Looks good, but clashes with the work around handler timeouts etc, so let's merge this after we have proper error and timeout handling

File: package.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
     "test": "ava test/*",
     "lint": "eslint --cache .",
     "checks": "npm test && npm run lint",
-    "refresh": "npm run clean && npm ci",
```
Comment (KnorpelSenf) [inline package.json:39]: Rationale?

File: src/composer.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ type GuardedContext<
  * that narrows down `tt.Update` when intersected with it.
  */
 type MountMap = {
-  [T in tt.UpdateType]: Record<T, object>
+  [T in tt.UpdateType]: Record<T, unknown>
```
Comment (KnorpelSenf) [inline src/composer.ts:58]: Less accurate and less readable. Any reason to do this?

File: src/telegraf.ts [hunk 3]
```diff
@@ -199,12 +205,10 @@ export class Telegraf<C extends Context = Context> extends Composer<C> {
 
   private botInfoCall?: Promise<tt.UserFromGetMe>
   async handleUpdate(update: tt.Update, webhookResponse?: http.ServerResponse) {
-    debug('Processing update', update.update_id)
```
Comment (KnorpelSenf) [inline src/telegraf.ts:202]: This has always been the most useful debug line for me. What's the rationale behind dropping it?

File: package.json (hunk 1 above)
Comment (wojpawlik) [reply package.json:39]: `npm ci` from within script fails on my machine, not willing to troubleshoot.

File: src/telegraf.ts (hunk 3 above)
Comment (wojpawlik) [reply src/telegraf.ts:202]: No reason, I'll leave it in then. I think it should be moved until after `getMe` call tho, do you agree?

File: package.json (hunk 1 above)
Comment (KnorpelSenf) [reply package.json:39]: And `npm i` does not?

File: src/composer.ts (hunk 2 above)
Comment (wojpawlik) [reply src/composer.ts:58]: Aesthetics. `T & unknown` is diplayed as `T`, leaving more space for all the union variants.

File: package.json (hunk 1 above)
Comment (KnorpelSenf) [reply package.json:39]: It works both on my machine and in CI. We can just as well leave it in. You don't need to use it.

File: src/telegraf.ts (hunk 3 above)
Comment (KnorpelSenf) [reply src/telegraf.ts:202]: Yes

File: package.json (hunk 1 above)
Comment (wojpawlik) [reply package.json:39]: I mean, `npm ci` works, but `npm ci` invoked by `npm run refresh` does not. Does that work for you?

Comment (KnorpelSenf) [reply package.json:39]: Sure, just tried it. I also don't know why there would be a difference. What Linux version does your installation of WSL rely on?

Comment (wojpawlik) [reply package.json:39]: I'm on Windows Windows, no WSL. And Node 14.0.0, I should probably upgrade. Re-adding then.

**Review by KnorpelSenf — APPROVED** [review]

**Pull request merged**